README.md

```diff
@@ -224,9 +224,44 @@ products_with_names_only = session.query({
 |`all`|Returns all documents from result set (considering `take` / `skip` options)|
 |`count`|Returns count of all documents matching query criteria (non-considering `take` / `skip` options)|
 
+## Working with secured server
+1. Put generated client certificate somewhere on your app structure. Optionally you can store it as string on your application config file or in the database
```

Comment:
That is a bad idea, let's get out of telling the user where to store the secret. Something like, use your organization policy toward certificate storage is must better and doesn't get us in hot waters

Reply:
ok i'll remove this from doc